LogMonitor/LogMonitorTests/LogFileMonitorTests.cpp

```diff
@@ -950,7 +950,7 @@ namespace LogMonitorTests
                 // It should the FULL content of the file, that's why we need
                 // to concatenate twice the content.
                 //
-                Assert::AreEqual((TO_WSTR(content) + TO_WSTR(content) + L"\n").c_str(), output.c_str());
+                Assert::IsTrue(output.find((TO_WSTR(content) + TO_WSTR(content) + L"\n").c_str()) != std::wstring::npos);

```

Comment:
why the change to `IsTrue`, do we still need the concatenation (+) then?

Reply:
This is because the stdout output will not only contain the concatenation of the contents but also the file name at the beginning. 

'AreEqual' at the point will always return false. However, checking if the output contains the concatenation should be true if the 'WriteToFile' function worked correctly.


![image](https://user-images.githubusercontent.com/28774968/195787937-32d4462a-d076-42f4-8471-87cc625cd8fb.png)

Reply:
got it, thx.